src/status_im/ui/screens/usage_data/events.cljs

```diff
@@ -5,8 +5,9 @@
 (handlers/register-handler-fx
   :help-improve-handler
   (fn [{db :db} [_ yes? address]]
-    (merge (when yes?
-             (accounts/account-update {:db db} {:sharing-usage-data? true}))
-           {:dispatch-n [(when yes? [:register-mixpanel-tracking address])
+    (merge (accounts/account-update {:db db} {:sharing-usage-data? yes?})
+           {:dispatch-n [(if yes?
+                           [:register-mixpanel-tracking address]
+                           [:unregister-mixpanel-tracking])
                          [:account-finalized]]})))
```

Comment:
it navigates a user to home, is it fine? because a user will expect a return to profile?

Reply:
Right, probably need to improve that

Reply:
This is expected to return a user to a profile